src/routes/user.rs

```diff
@@ -553,7 +556,15 @@ pub fn outbox(name: String, rockets: PlumeRocket) -> Option<ActivityStream<Order
     let user = User::find_by_fqn(&rockets, &name).ok()?;
     user.outbox(&*rockets.conn).ok()
 }
-
+#[get("/@/<name>/outbox?<page>")]
```

Comment:
does a blog also have such an outbox that could benefit from such a pagination?

Reply:
True enough, we'll get that made then.